### PR TITLE
Change the way KeysAuth and derivatives decide on keys on disk location

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

**Comment by chfast** [comment]
Side question: can we remove RSA-based manager?

**Comment by badb** [comment]
We may remove RSA or leave if it for test purpose. It was added to test if auth methods are general.

**Comment by chfast** [comment]
I will leave it then. Any comments about the code changes?

File: golem/core/keysauth.py [hunk 1]
```diff
@@ -131,6 +131,23 @@ def generate_new(self, difficulty):
         return
 
     @staticmethod
+    def __get_key_loc(file_name_pattern, uuid):
```
Comment (badb) [inline golem/core/keysauth.py]: Maybe we should set root_path as a variable in instance KeysAuth? It won't be static method, but use will be able to set path in config file.

**Pull request merged**